<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Planner</class>
 <widget class="QFrame" name="Planner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planner</string>
  </property>
  <widget class="QTabWidget" name="planner">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>431</width>
     <height>321</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>431</width>
     <height>321</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="day">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Day</string>
    </attribute>
    <widget class="QLabel" name="label_school">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>School</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Additional</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_task">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tasks</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_event">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Events</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="event"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="task_day"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="school_task"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="additional"/>
    </widget>
    <widget class="QPushButton" name="create_task">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create_task_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create_task_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create_my_event">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="week">
    <attribute name="title">
     <string>Week</string>
    </attribute>
    <widget class="QLabel" name="label_task_week">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tasks</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>160</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="task_week"/>
    </widget>
    <widget class="QPushButton" name="create_task_week">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>88</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create_event_week">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>88</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_event_week">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Events</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_events">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>211</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="school">
    <attribute name="title">
     <string>School</string>
    </attribute>
    <widget class="QLabel" name="label_shedule">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Shedule</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create_school_event">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_event_school">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ZOOM, Events</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>160</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="shedule">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QPushButton" name="change_shedule">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>88</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="event_school">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>191</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>View</string>
    </attribute>
    <widget class="QPushButton" name="open_txt">
     <property name="geometry">
      <rect>
       <x>287</x>
       <y>10</y>
       <width>131</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать документ</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>411</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="l_view"/>
    </widget>
    <widget class="QPushButton" name="open_img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать изображение</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
